### PR TITLE
[TaskManager] Minor bug fix

### PR DESCRIPTION
**Pull request by allisterakun**
Minor bug fixes for the following:
1. Task Manager does not dump the non-data pools after running the tasks.
2. Redundant logging in `handle_input_data_audit()`
### Context

Issue(s) closed by this pull request: closes #

- [x] The [changelog](https://docs.google.com/spreadsheets/d/1U-3igxdstHTFb6k5dVcE5-x9IL0r6q18eUiwwytKZyA/edit#gid=0) has been updated.
      
How big are the changes in the PR? Would you nominate it for a major version upgrade?
- [ ] Yes
- [x] No

### What

1. At the end of `TaskManager.start()` function, call `TaskManager.handle_post_processing()` to dump the logs, warning, and errors.
2. Removed the redundant logs in `TaskManager.handle_input_data_audit()`

### Why

To make sure we have logs, warnings, and errors for the TaskManager

### How

1. At the end of `TaskManager.start()` function, call `TaskManager.handle_post_processing()` to dump the logs, warning, and errors.
2. Removed the redundant logs in `TaskManager.handle_input_data_audit()`

### Test plan

No tests are needed.

### CODE REVIEW

File: RUFAS/task_manager.py [hunk 1]
```diff
@@ -134,6 +134,9 @@ def start(
         for i in range(len(runnable_args)):
             runnable_args[i]["task_id"] = f"{i+1}/{len(runnable_args)}"
         self._run_tasks(runnable_args, produce_graphics)
+        TaskManager.handle_post_processing(
+            args={}, input_manager=self.input_manager, output_manager=self.output_manager, task_id="TASK_MANAGER"
```
Comment (Isgotkowitz) [inline RUFAS/task_manager.py]: There are some arguments that have to be added here: "output_directory", "exclude_info_maps", and "variable_name_style". The first two are arguments to the `start` method so they can be set accordingly, but variable name style will have to be hardcoded.

**Review by Isgotkowitz — APPROVED** [review]
Some keys _must_ be added to the `args` argument for this to run properly, but after that it will be good to go.

File: RUFAS/task_manager.py (hunk 1 above)
Comment (ltp23) [reply RUFAS/task_manager.py]: This looks similar to the lines 103-112.

**Review by ltp23 — APPROVED** [review]
LGTM. Thanks Allister!

**Pull request merged**